integration_tests/commercial_power_plant_tests.feature

```diff
@@ -18,6 +18,6 @@ Feature: Commercial Power Plant tests
      Given we have a scenario named strategy_tests/commercial_producer_market_maker_rate
      And d3a is installed
      And we have a profile of market_maker_rate for strategy_tests.commercial_producer_market_maker_rate
-     When we run the d3a simulation with config parameters [1, 1] and strategy_tests.commercial_producer_market_maker_rate
-     Then we test that config parameters are correctly parsed for strategy_tests.commercial_producer_market_maker_rate [1, 1]
+     When we run the d3a simulation with config parameters [1, 0] and strategy_tests.commercial_producer_market_maker_rate
```

Comment:
Can you tell me the reason for this change or rather the reason for changing the iaa_fee to 0 in several integration tests?

Reply:
As discussed offline, reverting it